webextensions/api/cookies.json

```diff
@@ -494,6 +494,91 @@
                 "version_added": "15"
```

Comment:
Hi @jdatapple, can you clarify this "Only supports explicit." comment - neither MDN or Chrome document an `explicit` value for this type.

Reply:
@jdatapple are you able to look at this one?